libs/blocks/global-navigation/utilities/utilities.js

```diff
@@ -188,10 +188,13 @@ export function trigger({ element, event, type } = {}) {
 
 export const yieldToMain = () => new Promise((resolve) => { setTimeout(resolve, 0); });
 
-export const lanaLog = ({ message, e = '' }) => window.lana.log(`${message} ${e.reason || e.error || e.message || e}`, {
-  clientId: 'feds-milo',
-  sampleRate: 1,
-});
+export const lanaLog = ({ message, e = '' }) => {
+  const url = getMetadata('gnav-source');
+  window.lana.log(`${message} | gnav-source: ${url} | href: ${window.location.href} | ${e.reason || e.error || e.message || e}`, {
```

Comment:
As context for other reviewers - we're getting some odd messages from LANA, saying that an AEM page has Milo issues. This is why we're including the `window.location.href` part, to get some more details on what's going on.